bridges_add_old

```diff
@@ -96,15 +96,18 @@ if [ $exitstatus = 0 ]; then
           read -n 1 -s -r -p "Press any key to continue"
         else
           if [ "$MODE_BRIDGES" = "UseBridges 1" ]; then
-              bridge_address="$(<<< "$bridge_address" sed -e 's`obfs4`Bridge obfs4`g')"
-              bridge_address=$(echo -e "$bridge_address\n")
+            bridge_address="$(<<< "$bridge_address" sed -e 's`obfs4`Bridge obfs4`g')"
+            bridge_address=$(echo -e "$bridge_address\n")
           else
             bridge_address="$(<<< "$bridge_address" sed -e 's`obfs4`#Bridge obfs4`g')"
             bridge_address=$(echo -e "$bridge_address\n")
           fi
           echo -e "${RED}[+] Found a valid bridge!${NOCOLOR}"
           echo -e "${RED}[+] Saved a valid bridge!${NOCOLOR}"
           sudo echo $bridge_address >> $TORRC
+          sudo sed -i "s/^#UseBridges/UseBridges/g" ${TORRC}
+          sudo sed -i "s/^#UpdateBridgesFromAuthority/UpdateBridgesFromAuthority/g" ${TORRC}
+          sudo sed -i "s/^#ClientTransport/ClientTransport/g" ${TORRC}
```

Comment:
OBFS4 should only be activated with the menu entry "Activate OBFS4 and configured bridges". Adding additional bridges shouldn't activate/deactivate the OBFS4 status.

Reply:
answered here https://github.com/radio24/TorBox/pull/56#discussion_r648520279

Reply:
Also, this portion is inside the else (of the if statement), this else only occurs is there is new bridges added.
What I can do is the same as I did for the other scripts:
```
number_bridges
if number_total -gt 0 the Uncomment
```
This fits your criteria?

Reply:
Currently, no changes are needed. I only share my thoughts to be sure that I understand the code. It doesn't mean that something is wrong.